### DISCUSSION TITLE
다른 아이디의 빌드잇 맵을 월드로 가져올 수 있나요?

### DISCUSSION THREAD

**Discussion by runtostar** [naverz/zepeto-studio-kor#98] [5. Q&A]
수고 많으십니다. 빠른 업데이트 및 대응에 항상 감사드립니다.
빌드잇에서 생성한 맵을 월드에서 배경으로 사용할 경우, 현재는 동일 계정으로만 불러올 수 있는 것으로 파악됩니다.
혹시 다른 계정으로 만든 빌드잇 맵을 월드로 가져올 수 있는 방법이 있는지요?
현재는 없는 기능이라면, 향후 추가할 계획이 있는지요?
추가 계획이 있다면 시점은 대략 언제 정도가 될런지요?

**Comment by cheolhui-lee** [comment]
안녕하세요.
다른 계정의 맵은 저작권의 문제가 있기 때문에 불러오실 수 없습니다.
참고 부탁드립니다.
